core/src/main/kotlin/org/evomaster/core/Main.kt

```diff
@@ -596,7 +596,23 @@ class Main {
                     }
                 }
 
-            }else {
+            }else if (config.problemType == EMConfig.ProblemType.GRAPHQL) {
```

Comment:
why this? is it some merge issue? have u pulled latest from master?

Reply:
have pulled the latest one now.